### ISSUE TITLE
Personal Coaching

### ISSUE THREAD

**Issue by 7flash** [HighVibe/hvn#49]
**Outcome**
Implement feature that allows to schedule personal video coaching session with teacher, make payment, receive notification, and enable **encrypted** video meeting, provide all tools inside needed for personal coaching. 

**Purpose**
Instant progress and resolution of questions through individual connection with teacher

**Comment by JasonHighVibe** [comment]
Sounds great - Eventually could be good to provide a tool that allows Creators/Teachers to easily keep track of how many sessions and how much time he gives for his clients. i.e. if a teacher gives 1hr a week coaching session, he has an easy way to keep track of which clients he already has given coaching and maybe even notes to keep track of the session.

**Comment by 7flash** [comment]
Right, following features are needed for personal coaching: scheduling, time tracking, payments, challenges (coach can create tasks with monetary rewards or punishment)
